ini/ini_map.ts

```diff
@@ -185,10 +177,6 @@ export class IniMap {
     return size;
   }
 
-  get formatting(): Formatting {
-    return this.#formatting;
-  }
```

Comment:
This seemed exposed for testing purposes and I don't believe it made much sense otherwise. It seems we can remove it.